generator/src/googleapis/codegen/languages/php/default/templates/___api_className___/___models_className___.php.tmpl

```diff
@@ -53,9 +58,9 @@ class {{ model.className }} extends {% if model.superClass %}{{ model.superClass
     $this->{{ property.memberName }} = ${{ property.memberName }};
   }
 
-{% if property.typeHint %}
+{% if property.annotationType %}
   /**
-   * @return {{ property.typeHint }}{% if property.type == "array" or property.dataType == "map" %}[]{% endif %}
+   * @return {{ property.annotationType }}{% if property.type == "array" or property.dataType == "map" %}[]{% endif %}
```

Comment:
Why are we changing the existing `typeHInt` to `annotationType`? What is the difference? I thought the PR was only trying to add primitive types

Reply:
Because I thought it might be better to have a single concept to use in templates and keep the logic in the generator rather than here. This could be:

```
{% if property.typeHint %}
...
{% elif property.annotationType %}
...
{% endif %}
```

If you'd rather.

But I set `annotationType` for complex and scalar types https://github.com/googleapis/google-api-php-client-services/blob/5ec7b7858a8fc89e637ee8892d2ef535973b11b5/generator/src/googleapis/codegen/php_generator.py#L182-L192

If I had set `typeHint` direclty instead of introduce an annotation type, you'd scalar typehints everywhere which is a no go for PHP 5.6 support.